h/templates/client/about_this_version_dialog.html

```diff
@@ -0,0 +1,34 @@
+<div class="sheet">
+  <i class="close h-icon-close"
+    role="button"
+    title="Close"
+    ng-click="vm.onClose()"></i>
+  <div class="about-this-version">
+    <ul class="nav nav-tabs">
+      <li class="active"><a href="">About this version</a></li>
+    </ul>
+    <dl class="tab-content">
+      <dt class="about-this-version__key">Version: </dt>
+      <dd class="about-this-version__val">{{ vm.version }}</dd>
+      <dt class="about-this-version__key">User agent: </dt>
+      <dd class="about-this-version__val">{{ vm.userAgent }}</dd>
+      <dt class="about-this-version__key">URL: </dt>
+      <dd class="about-this-version__val">{{ vm.url }}</dd>
+      <div ng-if="vm.documentFingerprint">
+        <dt class="about-this-version__key">PDF fingerprint: </dt>
+        <dd class="about-this-version__val">{{ vm.documentFingerprint }}</dd>
+      </div>
+      <div ng-if="vm.auth.userid">
+        <dt class="about-this-version__key">Username: </dt>
+        <dd class="about-this-version__val">{{ vm.auth.username }}</dd>
+      </div>
+      <dt class="about-this-version__key">Date: </dt>
+      <dd class="about-this-version__val">{{ vm.dateTime | date:'dd MMM yyyy HH:mm:ss Z' }}</dd>
+      <div class="about-this-version-feedback">
+        <a class="about-this-version-feedback__link" href="mailto:support@hypothes.is?subject=Hypothesis%20feedback&amp;body=Version:%20{{ vm.version }}%0D%0AUser%20Agent:%20{{vm.userAgent}}%0D%0AURL:%20{{ vm.url }}%0D%0APDF%20fingerprint:%20{{ vm.documentFingerprint ? vm.documentFingerprint : '-' }}%0D%0AUsername:%20{{ vm.auth.username ? vm.auth.username : '-' }}%0D%0ADate:%20{{ vm.dateTime | date:'dd MMM yyyy HH:mm:ss Z' }} ">
```

Comment:
The support link is difficult to read in the template. I would suggest generating it via a helper function in the controller.